src/config/units.yaml

```
@@ -18,43 +18,116 @@ duration:
   description: refers to the duration of the observation
   unit: seconds
   aggregation: sum
-timestamp:
-  description: refers to the time of occurrence of the observation
-  unit: RFC3339
-  aggregation: None
-carbon:
-  description: an amount of carbon emitted into the atmosphere
-  unit: gCO2e
-  aggregation: sum
-energy:
-  description: amount of energy utilised by the component
-  unit: kWh
-  aggregation: sum
 e-cpu:
   description: Energy consumed by the CPU of the component
   unit: kWh
   aggregation: sum
+el:
+  description: expected lifespan for some component
+  unit: seconds
+  aggregation: none
 e-mem:
```

```suggestion
energy-memory:
```

Spirit of using long descriptive names